src/index.js

```diff
@@ -99,6 +99,10 @@ class Superlogin extends EventEmitter {
 			return Promise.reject(response);
 		};
 
+		// clear interceptors from a previous configure call
+		this._http.interceptors.request.handlers = [];
```

Comment:
Using `axios.interceptors.request.eject` would make more sense to me.
But you'll need to do more refactoring.